dbms/src/Storages/DeltaMerge/tests/gtest_dm_file.cpp

```diff
@@ -415,9 +415,8 @@ try
                 files.insert(itr.name());
             }
         }
-        ASSERT_EQ(files.size(), 3); // handle data / col data and merged file
+        ASSERT_EQ(files.size(), 2); // col data and merged file
```

Comment:
why handle data is not exist anymore?

Reply:
The data size is smaller, so merged into the merge file.

Reply:
Get it.
Should update the comment here https://github.com/pingcap/tiflash/blob/17effd924cc8a95d31c0481c5945166323279a4b/dbms/src/Storages/DeltaMerge/tests/gtest_dm_file.cpp#L365-L367